pkgs/tools/misc/byobu/default.nix

```diff
@@ -11,26 +16,44 @@ stdenv.mkDerivation rec {
 
   doCheck = true;
 
-  buildInputs = [ python3 perl ];
-  propagatedBuildInputs = [ textual-window-manager ];
-
-  meta = {
+  buildInputs = [ perl pythonEnv makeWrapper vim bc gettext];
+  propagatedBuildInputs = [ textual-window-manager screen pythonEnv ];
+
+  postUnpack = ''
+    substituteInPlace $sourceRoot/usr/bin/byobu-select-profile.in \
+      --replace "/bin/true" "true" \
+      --replace "gettext" "${gettext}/bin/gettext"
+    substituteInPlace $sourceRoot/usr/bin/byobu-export.in \
+      --replace "gettext" "${gettext}/bin/gettext"
+  '';
+
+  postInstall = ''
+    wrapProgram $out/bin/byobu-status-detail --prefix PATH : ${vim}/bin/
+    wrapProgram $out/bin/byobu-ulevel --prefix PATH : ${bc}/bin/
+    wrapProgram $out/bin/byobu-config --set BYOBU_PYTHON ${pythonEnv}/bin/python
+    #for i in $out/bin/*; do
+    #  wrapProgram "$i" --set BYOBU_PYTHON "${pythonEnv}/bin/python" \
+    #    --prefix PATH : ${binPath}
+    #done
```

Comment:
is this still needed? it's commented out

Reply:
See comment below